include/constants/species.h

```diff
@@ -685,11 +685,11 @@
 #define SPECIES_FLETCHLING                              661
 #define SPECIES_FLETCHINDER                             662
 #define SPECIES_TALONFLAME                              663
-#define SPECIES_SCATTERBUG                              SPECIES_SCATTERBUG_ICY_SNOW
+#define SPECIES_SCATTERBUG                              GetSpeciesPreEvolution(SPECIES_SPEWPA)
```

Comment:
What happens if you use `givemon SPECIES_SCATTERBUG, 5` in a script?

Reply:
Pretty sure this won't compile

Reply:
oh, is that so
guess I will have to do something else then

Reply:
This config seems redundant to me. The hacker can already choose any scatterbug/spewpa/vivillion in their encounter tables.

Reply:
> This config seems redundant to me. The hacker can already choose any scatterbug/spewpa/vivillion in their encounter tables.

This decides what Vivillon form eggs hatch into, since all of them only hatch into one form in the actual games.

Reply:
the why do we have 10 scatterbugs?

Reply:
What determines the evolution officially is the "real world location" data in their saved data. This is an inmutable value that sticks regardless if it's traded to a different game. Since we don't have that field, we have that as being part of our "species ID".